beliefs/allTrustBeliefs.csv

```diff
@@ -1,2 +1,6 @@
-name;competence;willingness;ticks_played
-;;;;
\ No newline at end of file
+name;competence;willingness
+alice;0.64320095454004325;0.754345553323409109509
```

Comment:
Make sure you all remove the ticks_played column when we merge this so stuff doesn't break

Reply:
i don't know how i feel about having values in the csvs we upload, shouldn't we want those locally instead?

Reply:
removed